### ISSUE TITLE
Slideshow Block: Allow choosing a background color/full transparency

### ISSUE THREAD

**Issue by eduardozulian** [Automattic/jetpack#13976]
#### Is your feature request related to a problem? Please describe.
The default gray color sometimes does not match the site's colors. It would be great if we offered a way to:

1. Add a custom background to the slideshow
2. Make the slideshow transparent

This, connected with #12178, produces a not ideal layout for the slideshow.

#### Describe alternatives you've considered
For now, CSS is the way to go. Here's how to make it transparent:

```
.wp-block-jetpack-slideshow .wp-block-jetpack-slideshow_slide {
  background: transparent !important;
}
```

If would be great not to rely on CSS for that.

Thanks for considering!

**Comment by TeniCola** [comment]
Adding another case where a user wanted to remove the gray background from their slideshow block, in 16816788-hc. I offered this CSS as a work around to help the user.

**Comment by eduardozulian** [comment]
16587612-hc

**Comment by AMcGrady** [comment]
Also reported here: #17570511-hc

**Comment by eamcgill90** [comment]
Reported here: 19284805-hc

**Comment by mikeicode** [comment]
Reported 3135590-zen

**Comment by kspilarski** [comment]
Also reported on 22759329-hc

**Comment by eduardozulian** [comment]
Another: 22979571-hc

**Comment by aamills** [comment]
Spoke with a user today that would like to have the option to change the background color of this block. 23137986-hc

Gave CSS as a work around.

**Comment by katiebethbrown** [comment]
Another report today in 23234064-hc. Shared the CSS:

**Comment by rachelwinspear** [comment]
Another report #24146209-hc - Shared CSS

**Comment by sourourn** [comment]
Another report here: #24295277-hc

**Comment by jeherve** [comment]
This was fixed in #17250; the slideshow background will be transparent starting in Jetpack 9.0.

**Comment by iamtakashi** [comment]
Thanks for removing the background. There is still a background colour appearing if I choose "fade" transition though. Can we remove it also?

Editor:
![Screenshot 2020-10-05 at 22 20 49](https://user-images.githubusercontent.com/908665/95133608-7a48b080-0759-11eb-927a-aa91d01735d7.jpg)
 
Frontend:
![Screenshot 2020-10-05 at 22 27 04](https://user-images.githubusercontent.com/908665/95133934-f511cb80-0759-11eb-9cc5-0414ef65a8f9.jpg)
